bucket/trafficmonitor.json

```diff
@@ -14,7 +16,27 @@
         }
     },
     "extract_dir": "TrafficMonitor",
-    "bin": "TrafficMonitor.exe",
+    "installer": {
+        "script": [
+            "if (!(Test-Path \"$persist_dir\\config.ini\")) { Set-Content \"$dir\\config.ini\" @('[general]', 'check_update_when_start = false') }",
+            "Set-Content \"$dir\\global_cfg.ini\" @('[config]', 'portable_mode = true')"
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "if (!(Test-Path \"$persist_dir\")) {",
+            "   New-Item \"$persist_dir\" -ItemType Directory | Out-Null",
+            "   Copy-Item \"$dir\\config.ini\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force",
+            "}"
+        ]
+    },
```

Comment:
Why?

Reply:
This will never work.

Reply:
Flow of potential update/installation

1. User will execute update (force update)
1. Old manifest will be loaded
	1. No uninstaller.script property defined in old manifest
1. User will loose configuration
	1. You will not copy anything
1. New version installed
1. User will update
1. This will never be executed.

Reply:
It will execute once when people who has already installed this app update their manifest since the old one does not persist anything.